### PR TITLE
Fix sonar to reflect datastruct has no default

### PR DESCRIPTION
**Pull request by HThuren**
fix that sonarElements has no default, ie condition must be x > 0

### CODE REVIEW

File: src/js/tabs/setup.js
Comment (haslinghuis) [inline src/js/tabs/setup.js:410]: This should work too:

```suggestion
                    } else if (have_sensor(FC.CONFIG.activeSensors, "sonar") && FC.SENSOR_CONFIG_ACTIVE.sonar_hardware) {
```

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by HThuren** [comment]
@haslinghuis - got a mail regarding you suggestion to code change, but not here ind github (??)

**Comment by haslinghuis** [comment]
All good & ready to go

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
